.pre-commit-config.yaml

```diff
@@ -1,21 +1,22 @@
 repos:
   - repo: local
     hooks:
-      - id: ruff
-        name: ruff
+      - id: ruff-lint
+        name: ruff-lint
         entry: ruff check .
         args: [ --config, pyproject.toml, --fix, --show-fixes, --exit-non-zero-on-fix ]
         language: system
         types_or: [python]
         require_serial: true # run once for all files
         pass_filenames: false
-      - id: black
-        name: black
-        entry: black
+      - id: ruff-format
+        name: ruff-format
+        entry: ruff format .
+        args: [ --config, pyproject.toml ]
         language: system
-        types_or: [python, pyi]
```

Comment:
why did you change it and what's the difference to `types`?

Reply:
Because we don't have `pyi` files. Also `pyright` already isn't looking at those, so I thought that we don't need it.

I could add `pyi` to both instead?

`types_or` works with an `OR`, so if any of the tags is present, the file is accepted.
`types` uses `AND`, so all tags need to be present in the same file

Reply:
thanks for the explanation. I agree with removing `pyi`, we currently don't need it.